draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -999,15 +1008,17 @@ The following is a brief definition of elliptic curves, with an emphasis on
 important parameters and their relation to hashing to curves.
 For further reference on elliptic curves, consult {{CFADLNV05}} or {{W08}}.
 
-Let F be the finite field GF(q) of prime characteristic p. In most cases F
-is a prime field, so q = p. Otherwise, F is a field extension, so q = p^m for
-an integer m > 1. This document writes elements of field extensions
+Let F be the finite field GF(q) of prime characteristic p > 3.
+(This document does not consider elliptic curves over fields of characteristic 2 or 3.)
+In most cases F is a prime field, so q = p.
+Otherwise, F is a field extension, so q = p^m for an integer m > 1.
```

Comment:
These break changes done in #193 .

Reply:
Thanks for the heads-up! I made sure to keep #193's changes when rebasing.